common/utils.go

```diff
@@ -549,3 +550,31 @@ func GetAllRingFileMd5s() (map[string]string, error) {
 		return ringMap, err
 	}
 }
+
+func IsCorruptDBError(err error) bool {
+	a := err.Error()
+	for _, b := range []string{
+		"database disk image is malformed",
+		"file is encrypted or is not a database",
+		"file is not a database",
+	} {
+		if strings.Contains(a, b) {
+			return true
+		}
+	}
+	return false
+}
+
+func QuarantineFile(filePath string) error {
+	f, err := ioutil.TempFile(path.Dir(filePath), "quarantined-")
+	if err != nil {
+		return fmt.Errorf("could not quarantine; TempFile error: %s", err)
+	}
+	toFilePath := f.Name()
```

Comment:
is it weird to do them in place? for example- recon has a quarantine call

`
-q    Get cluster quarantine stats
`

that shows the number of quarantined stuff out on nodes. we wouldn't be able to get that for containers anymore right? do people use those to help see if drives are broke?

Reply:
Gotcha. Sounds like a good plan to change from what I have. As far as anymore, hummingbird didn't quarantine dbs at all before, so... ;) I'll peek at recon and make sure this lines up with that. Also, you mentioned elsewhere that this should be quarantining the whole dir, so I'll fix that too.